apps/ledger-live-mobile/src/components/RootNavigator/EthereumEditTransactionNavigator.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useMemo } from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTranslation } from "react-i18next";
+import { useTheme } from "styled-components/native";
+
+import { ScreenName } from "../../const";
+import { getStackNavigatorConfig } from "../../navigation/navigatorConfig";
+import StepHeader from "../StepHeader";
+import { MethodSelection } from "../../families/ethereum/EditTransaction/MethodSelection";
+import { EthereumEditTransactionParamList } from "./types/EthereumEditTransactionNavigator";
+import { SpeedupTransaction } from "../../families/ethereum/EditTransaction/Speedup";
+import { CancelTransaction } from "../../families/ethereum/EditTransaction/Cancel";
+import SendSummary from "../../screens/SendFunds/04-Summary";
+import SelectDevice from "../../screens/SelectDevice";
+import SendConnectDevice from "../../screens/ConnectDevice";
+import SendValidationSuccess from "../../screens/SendFunds/07-ValidationSuccess";
+import SendValidationError from "../../screens/SendFunds/07-ValidationError";
+
+const Stack = createStackNavigator<EthereumEditTransactionParamList>();
+
+export default function EditTransactionNavigator() {
```

Comment:
This creates the EditTransactionNavigator with all the screens of the workflow. It's quite similar to the SendFundsNavigator